scripts/basic-workflow-tests.sh

```diff
@@ -277,10 +266,21 @@ test_train() {
             TRAIN_ARGS+=("--gguf-model-path" "${DATA_HOME}/instructlab/models/granite-7b-lab-Q4_K_M.gguf")
         fi
 
+        ilab model train "${TRAIN_ARGS[@]}"
+    else
+        # TODO Only cuda for now
+        TRAIN_ARGS+=("--legacy" "--device=cuda")
+        if [ "$FULLTRAIN" -eq 0 ]; then
+            TRAIN_ARGS+=("--4-bit-quant")
+        fi
```

Comment:
Looking at the `-f` flag I'm unclear as to how this and the new `-T` flag differ - is `-f` just using legacy training with safetensors?

Reply:
* `-T` -- training lib
* `-f` -- `--legacy`
* neither option -- `--legacy --4-bit-quant`